Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -1757,6 +1839,113 @@ def get_audit_firewall_container_alerts(client: PrismaCloudComputeClient, args:
     )
 
 
+def get_alert_profiles_command(client, args):
+    """
+    Get the alert profiles.
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-get-alert-profiles command arguments
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    project = args.get("project", None)
+    response = client.get_alert_profiles_request(project)
+    return CommandResults(
+        outputs_prefix='PrismaCloudCompute.AlertProfiles',
+        outputs_key_field='_Id',
+        outputs=format_context(response),
+        raw_response=response
+    )
+
+
+def get_settings_defender_command(client, args):
+    """
+    Get the defender settings.
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-get-settings-defender command arguments
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    response = client.get_settings_defender_request()
```

Comment:
it looks like there is always defender settings as long as there is a defender even if not online.  I am using the onebox deployment so there is always an associated defender and this will pull it as being inactive but still pull the default settings for a defender.